### DISCUSSION TITLE
Help:  ERR_PNPM_WORKSPACE_PKG_NOT_FOUND  In : "@logto/cli@workspace:^1.1.0" is in the dependencies but no package named "@logto/cli" is present in the workspace

### DISCUSSION THREAD

**Discussion by jobayer977** [logto-io/logto#3951] [Q&A] [answered]
### Describe the bug
When I try to install using pnpm install

### Expected behavior
I think should install packages

### How to reproduce?
npm init @logto@latest
pnpm install

### Context
{
  "name": "@logto/root",
  "private": true,
  "license": "MPL-2.0",
  "type": "module",
  "scripts": {
    "preinstall": "npx only-allow pnpm",
    "pnpm:devPreinstall": "cd packages/connectors && node templates/sync-preset.js",
    "prepare": "if test \"$NODE_ENV\" != \"production\" && test \"$CI\" != \"true\" ; then husky install ; fi",
    "prepack": "pnpm -r prepack",
    "dev": "pnpm -r prepack && pnpm start:dev",
    "dev:cloud": "pnpm -r prepack && pnpm start:dev:cloud",
    "start:dev": "pnpm -r --parallel --filter=!@logto/integration-tests --filter \"!./packages/connectors/connector-*\" --filter=!@logto/cloud dev",
    "start:dev:cloud": "CONSOLE_PUBLIC_URL=/ IS_CLOUD=1 pnpm -r --parallel --filter=!@logto/integration-tests --filter \"!./packages/connectors/connector-*\" dev",
    "start": "cd packages/core && NODE_ENV=production node .",
    "start:cloud": "cd packages/cloud && NODE_ENV=production node .",
    "cli": "logto",
    "alteration": "logto db alt",
    "connectors:build": "pnpm -r --filter \"./packages/connectors/connector-*\" build",
    "//": "# `changeset version` won't run version lifecycle scripts, see https://github.com/changesets/changesets/issues/860",
    "ci:version": "changeset version && pnpm -r version",
    "ci:build": "pnpm -r build",
    "ci:lint": "pnpm -r --parallel --workspace-concurrency=0 lint",
    "ci:stylelint": "pnpm -r --parallel --workspace-concurrency=0 stylelint",
    "ci:test": "pnpm -r --parallel --workspace-concurrency=0 test:ci"
  },
  "devDependencies": {
    "@changesets/cli": "^2.25.0",
    "@commitlint/cli": "^17.0.0",
    "@commitlint/config-conventional": "^17.0.0",
    "@commitlint/types": "^17.0.0",
    "@types/pg": "^8.6.6",
    "husky": "^8.0.0",
    "pg": "^8.8.0",
    "typescript": "^5.0.0"
  },
  "engines": {
    "node": "^18.12.0",
    "pnpm": "^8.0.0"
  },
  "alias": {
    "html-parse-stringify": "html-parse-stringify/dist/html-parse-stringify.module.js",
    "react-hook-form": "react-hook-form/dist/index.esm.mjs",
    "superstruct": "superstruct/lib/index.es.js"
  },
  "pnpm": {
    "peerDependencyRules": {
      "allowedVersions": {
        "react": "^18.0.0",
        "jest": "^29.1.2",
        "stylelint": "^15.0.0"
      }
    }
  },
  "dependencies": {
    "@logto/cli": "workspace:^1.1.0"
  }
}

- [ ] Logto Cloud
- [ ] Self-hosted, Logto version = 
  - [ ] Container (Docker image)
  - [x] Raw Node.js

### Screenshots
<img width="948" alt="image" src="https://github.com/logto-io/logto/assets/34404032/44ce38e2-ffeb-4165-8191-18cafe32e861">

**Comment by gao-sun** [comment, marked as answer]
you don't need to run `pnpm install` when you use `npm init @logto@latest`. btw the issues is not support channel, i'll convert this into discussion

**Reply by terry-fei** [reply]
it seems break docker build, because Dockerfile RUN pnpm i
https://github.com/logto-io/logto/blob/master/Dockerfile#L18
